src/routes/Project/actions/ClipActionHandlers.js

```diff
@@ -0,0 +1,13 @@
+import ProjectConstants from '../constants';
+
+const ACTION_HANDLERS = {
+  [ProjectConstants.ADD_CLIP]: (state, action) => {
+    const id = action.payload.id;
+    const track = action.payload.start.y;
+    const startTime = action.payload.start.x;
+    const endTime = action.payload.end.x;
+    return { ...state,  [id]: { track, startTime, endTime } };
```

Comment:
as i understand it, the state variable coming into this function is the subset of the state concerning clips only, correct?